python/lsst/ts/m2/model.py

```diff
@@ -0,0 +1,849 @@
+
+        According with the vendor documents these limits are hard-coded in
+        the real system, so they are also hard-coded in the simulator.
+        """
+        return 444.82
+
+    @property
+    def max_tangent_force(self):
+        """Tangent force limit (in Newtons).
+
+        According with the vendor documents these limits are hard-coded in
+        the real system, so they are also hard-coded in the simulator.
+        """
+        return 4893.04
+
+    @property
+    def position_limit_radial(self):
+        """Mirror radial motion limit (in mm)."""
+        return 7.74
+
+    @property
+    def position_limit_z(self):
+        """Mirror motion limit in the optical direction (in mm)."""
+        return 7.89
+
+    def configure(self, config_dir, lut_path):
+        """Do the configuration.
+
+        Parameters
+        ----------
+        config_dir : `pathlib.PosixPath`
+            Configuration directory.
+        lut_path : `str`
+            Look-up table (LUT) path.
+        """
+
+        # Look-up table (LUT) factors (filename, basically).
+        lut_factors = set(
+            [
+                "F_0.csv",
+                "F_A.csv",
+                "F_E.csv",
+                "F_F.csv",
+                "Tr.csv",
+                "Tu.csv",
+                "Tx.csv",
+                "Ty.csv",
+                "temp_inv.txt",
+            ]
+        )
+
+        for lut in lut_factors:
+            self.log.debug(f"Reading {lut}...")
+            name, ext = lut.split(".")
+            if ext == "csv":
+                data = pd.read_csv(
+                    config_dir / lut_path / lut,
+                    header="infer" if name.startswith("F") else None,
+                )
+                self.lut[name] = np.float64(data)
+                if name == "F_E":
+                    self.lut["lutInAngle"] = np.float64(data.keys())
+            else:
+                self.lut[name] = np.loadtxt(config_dir / lut_path / lut)
+
+    def apply_forces(self, force_axial, force_tangent):
+        """Apply force.
+
+        Parameters
+        ----------
+        force_axial : `list` or `numpy.ndarray`
+            Force of the axial actuators in Newton.
+        force_tangent : `list` or `numpy.ndarray`
+            Force of the tangent actuators in Newton.
+        """
+
+        # Check limits
+        self.check_axial_force_limit(apply=force_axial)
+        self.check_tangent_force_limit(apply=force_tangent)
+
+        self.axial_forces["applied"] = np.array(force_axial)
+        self.tangent_forces["applied"] = np.array(force_tangent)
+
+    def check_axial_force_limit(self, apply=None):
+        """Check if axial forces are out of bounds.
+
+        Parameters
+        ----------
+        apply : `numpy.ndarray` or `None`, optional
+            Forces to apply in Newton. If `None` use current setup. (default is
+            None.)
+
+        Returns
+        -------
+        `numpy.ndarray`
```

Comment:
As I mentioned before, consider adding the name of the variable you are returning:

```
Returns
--------
demanded_axial_force : `numpy.ndarray`
```

Reply:
Done in the commit: f20e3d7. Thanks!